staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -80,6 +88,12 @@ func (s *DelegatingAuthorizationOptions) ApplyTo(c *server.Config) error {
 		return nil
 	}
 
+	if s.SkipInClusterLookup {
+		// for local development only allow system:master group, reject everybody else.
+		c.Authorizer = union.New(authorizerfactory.NewPrivilegedGroups("system:master"), authorizerfactory.NewAlwaysDenyAuthorizer())
+		return nil
```

Comment:
What was happening before if no delegation config was given? I would have expected us to already end up with a union between the privileged authorizer and a deny all authorizer. Also, the group is `system:masters`

Reply:
Rather than a separate flag to control this, the authorizer setup should maybe just tolerate missing config in more cases by returning a deny all authorizer that gets unioned with the privileged one